django/db/backends/base/schema.py

```diff
@@ -99,6 +99,8 @@ def execute(self, sql, params=()):
         Executes the given SQL statement, with optional parameters.
         """
         # Log the command we're running, then run it
+        if isinstance(sql, tuple):
+            sql, params = sql  # Used by e.g. _create_index_sql()
```

Comment:
The `params` argument is ignored? I guess it might be appropriate to raise an error if both are provided to prevent obscure issue if this cannot be done better.